src/ansys_sphinx_theme/theme/ansys_sphinx_theme/_templates/autoapi/python/module.rst

```diff
@@ -29,7 +29,7 @@
    :hidden:
 
     {% for obj in objects_list %}
-    {{ icon }} {{ obj.short_name }}<{{ obj.include_path }}>
+    <i class="{{ icon }}"></i> {{ obj.short_name }}<{{ obj.include_path }}>
```

Comment:
```suggestion
    <i class="{{ icon }}" /> {{ obj.short_name }}<{{ obj.include_path }}>
```

Reply:
Also, is there any way the `<` and `>` do not render so we can avoid replacing the `&lt` and `&gt` with the actual symbols? What about escaping these?

Reply:
> Also, is there any way the `<` and `>` do not render so we can avoid replacing the `&lt` and `&gt` with the actual symbols? What about escaping these?

I'm testing it out and looking into other options too. I'll let you know if I find a solution.